src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -591,7 +591,33 @@ private function getAdjacentRecordID($offset)
 
         $map = $list->limit($limit, $limitOffset)->column('ID');
         $index = array_search($this->record->ID, $map ?? []);
-        return isset($map[$index+$offset]) ? $map[$index+$offset] : false;
+        $position = $index+$offset;
+        $page = $currentPage;
+        $lastPosition = count($map) - 1;
+        $hasPages = $this->getNumPages($gridField) > $currentPage;
```

Comment:
```suggestion
        $hasMorePages = $this->getNumPages($gridField) > $currentPage;
```
The variable isn't "this has pages", but "this has _more_ pages after the current one"

Reply:
DONE